sys-apps/pacman/pacman-5.0.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+
+DESCRIPTION="Archlinux's binary package manager"
+HOMEPAGE="https://archlinux.org/pacman/"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.archlinux.org/pacman.git"
+	EGIT_BRANCH="master"
```

Comment:
I thought of easier maintenance if the branch might be changed later. Removed it locally (git push pending).